prac_05/hex_colours.py

```diff
@@ -0,0 +1,12 @@
+COLOUR_DICTIONARY = {"AliceBlue": "#f0f8ff", "black": "#000000", "brown": "	#a52a2a", "chocolate": "#d2691e", "coral": "#ff7f50",
+                     "CornflowerBlue": "#6495ed", "DarkKhaki": "#bdb76b", "aquamarine2": "#76eec6", "linen": "#faf0e6", "plum": "#dda0dd"}
+
+
```

Comment:
Adding a print statement so that the user knows what options he has to choose from would be helpful.